swift_browser_ui_frontend/src/components/ShareModal.vue

```diff
@@ -1,67 +1,74 @@
 <template>
   <c-card class="share-card">
-    <header>
-      <h4 class="title is-4 has-text-dark">
+    <div class="heading">
```

Comment:
should we use `<c-card-title>` ?

and to encapsulate the `c-button` to use `<c-card-actions>` ?

Reply:
I also wanted to use `<c-card-title>` in the first place, but it has `<c-title>` underneath and this one has an undesirable underline which we don't have in the current Figma design. 

Using `<c-card-actions>` is a better idea 👍 . I will push the change for it.